src/pyspiffe/workloadapi/default_workload_api_client.py

```diff
@@ -139,24 +140,24 @@ def fetch_x509_bundles(self) -> X509BundleSet:
         return self._create_bundle_set(response.bundles)
 
     def fetch_jwt_svid(
-        self, audiences: List[str], subject: Optional[str] = None
+        self, audiences: Set[str], subject: Optional[SpiffeId] = None
     ) -> JwtSvid:
         """Fetches a SPIFFE JWT-SVID.
 
         Args:
-            audiences: List of audiences for the JWT SVID.
-            subject: SPIFFE ID subject for the JWT.
+            audiences: Set of audiences for the JWT SVID.
+            subject: SpiffeID subject for the JWT.
```

Comment:
nit: `SpiffeId`

Reply:
For documentation purposes I think we should follow the same approach used on the [SPIFFE website](https://spiffe.io/) and the [GitHub repo](https://github.com/spiffe/spiffe). In both places it is used as `SPIFFE ID`.
What do you think?

Reply:
makes sense, `SPIFFE ID` is fine

Reply:
As a separate task it is worth looking for other places where we use the type name `SpiffeId` in the documentation, to keep consistency.

Reply:
Fine, I've changed to SPIFFE ID where it concerns to this PR.